src/PokemonTrainer.java

```diff
@@ -0,0 +1,26 @@
+import java.util.List;
+
+public class PokemonTrainer {
+    String  name;
+    List<Pokemon> pokemons;
+
+    PokemonTrainer(String name, List<Pokemon> pokemons) {
```

Comment:
Je constructors mogen public zijn. Nu zijn ze niks, wat "package private" betekend. Je hebt nu maar 1 package, namelijk "src" (of eigenlijk is "src" de module en heeft java een onzichtbare, naamloze package gegenereerd, maar dat mag je vergeten), maar straks in de SpringBoot cursus gaan we wel verschillende packages gebruiken. Wanneer jou constructors dan "package private" zijn, kun je ze niet overal gebruiken.